test/patches.jl

```diff
@@ -23,6 +23,7 @@ git_push_patch = @patch function CompatHelper.git_push(
 end
 git_gmb_patch = @patch CompatHelper.git_get_master_branch(master_branch) = nothing
 git_checkout_patch = @patch CompatHelper.git_checkout(branch) = nothing
+git_remote_add_and_seturl_patch = @patch CompatHelper.git_remote_add_and_seturl(remote_name, url) = nothing
```

Comment:
What is the value of `remote_name` here?

Reply:
It'd be `compathelper-ssh-remote`, though actually I realise this isn't needed at all, it's not being used anywhere.